nbbuild/cluster.properties

```diff
@@ -975,6 +975,11 @@ nb.cluster.groovy.depends=\
         nb.cluster.extide,\
         nb.cluster.java
 nb.cluster.groovy=\
+        gradle,\
+        gradle.java,\
+        gradle.persistence,\
+        gradle.spring,\
+        gradle.test,\
         groovy.antproject,\
         groovy.editor,\
         groovy.gsp,\
```

Comment:
Also change the `javadoc` config and make sure `ant build-javadoc` includes the new modules with APIs.

Reply:
Ok. Found how to do it. It will be added.